icu4c/source/data/curr/ar.txt

```diff
@@ -763,9 +763,13 @@ ar{
             "SKK",
             "كرونة سلوفاكية",
         }
+        SLE{
+            "SLE",
+            "ليون سيراليوني",
+        }
         SLL{
             "SLL",
-            "ليون سيراليوني",
+            "ليون سيراليوني - 1964-2022",
```

Comment:
Just FYI:
I just realized that we occasionally have hardcoded numbers like this in display names. This is a bit problematic in locales where we can expect users to select different numbering systems (in this case Latn vs. Arab).

Reply:
Sounds like a godd topic for a CLDR ticket... thanks!